frontend/src/components/dashboard/index.tsx

```diff
@@ -21,23 +21,21 @@ const DashBoard = () => {
     const projectname = localStorage.getItem('projectName')
     const navigate = useNavigate()
     const handleNavigate = (value: string) => {
-        const projectId = localStorage.getItem('projectId')
-
         switch (value) {
             case 'model':
-                navigate(`/projects/${projectId}/models`)
+                navigate(`/project/models`)
```

Comment:
This exactly explains why we would all routes like these in a single place, so that we don't have to update them one by one